jerry-debugger/jerry-client-ws.py

```diff
@@ -250,6 +256,28 @@ def do_delete(self, args):
             else:
                 print("Error: Breakpoint %d not found" % (breakpoint_index))
 
+    def do_pendingdel(self, args):
+        """ Delete the given pending breakpoints from the list """
+        if not args:
+            print("Error: Pending breakpoint index expected")
+            return
+
+        else:
+            try:
+                breakpoint_index = int(args)
+            except ValueError as val_errno:
+                print("Error: Integer number expectes, %s" % (val_errno))
+                return
+
+            if breakpoint_index <= len(self.debugger.pending_breakpoint_list):
```

Comment:
I don't understand this. Normal and pending breakpoints should share the breakpoint counter.